etc/icinga2/scripts/mail-service-notification.sh

```diff
@@ -79,6 +81,8 @@ do
     i) ICINGAWEB2URL=$OPTARG ;;
     l) HOSTNAME=$OPTARG ;; # required
     n) HOSTDISPLAYNAME=$OPTARG ;; # required
+    X) HOSTNOTES="${OPTARG}" ;;
+    x) SERVICENOTES="${OPTARG}" ;;
```

Comment:
It won't hurt as you did, but _consider_ writing the assignment like the others. See also:

https://unix.stackexchange.com/questions/97560/are-quotes-needed-for-local-variable-assignment

Reply:
Sure, no problem. 
I always use quotes and curly brackets in bash, as for me it is no disadvantage but rather a security advantage (I know, not necessary in most cases, but it doesn't hurt imho :))
Ref, which I always like to read: https://unix.stackexchange.com/questions/171346/#171347

I actually wondered why the notification.sh-Scripts are not written with  brackets and stuff.